assembly/src/parsers/mod.rs

```diff
@@ -14,6 +14,9 @@ use context::ParserContext;
 mod adv_ops;
 mod field_ops;
 mod io_ops;
+mod labels;
+use labels::CONSTANT_LABEL_PARSER;
+pub use labels::PROCEDURE_LABEL_PARSER;
 mod serde;
 mod stack_ops;
 mod u32_ops;
```

Comment:
minor nit: I'd probably put label imports about this group of imports. For example:
```Rust
mod labels;
use labels::CONSTANT_LABEL_PARSER;
pub use labels::PROCEDURE_LABEL_PARSER;

mod adv_ops;
mod field_ops;
mod io_ops;
mod serde;
mod stack_ops;
mod u32_ops;
```